angular.json

```diff
@@ -15,13 +15,16 @@
       "prefix": "app",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-devkit/build-angular:application",
```

Comment:
In this file broadly I tried to reintroduce as few of our previous customisations as I could without things breaking, as I wasn't really sure which if any were deliberate adjustments to the old build's boilerplate.